src/ios/AppsFlyerPlugin.m

```diff
@@ -193,7 +193,7 @@ -(void)onConversionDataRequestFailure:(NSError *) _errorMessage {
     NSDictionary* errorMessage = @{
                                    @"status": afFailure,
                                    @"type": afOnInstallConversionFailure,
-                                   @"data": _errorMessage
+                                   @"data": _errorMessage..localizedDescription
```

Comment:
@dasMajo is the ..localizedDescripton (double dot) a typo? If so, could you please submit a corrected pull request?